core/app/crash/reporting/encoder.go

```diff
@@ -18,48 +18,60 @@ import (
 	"bytes"
 	"io"
 	"mime/multipart"
+
+	"github.com/google/gapid/core/app/crash"
```

Comment:
I'm not convinced that using the `crash` package here is better than no intra-dependencies.